src/routes/account/[uid]/+page.server.ts

```diff
@@ -41,6 +42,16 @@ export const actions = {
             }
         }
         console.log("Creating Stripe session");
+
+        const subscribedBefore = await hasCustomerSubscribedBefore(customer);
+        var subscriptionData;
+        if (subscribedBefore) {
+            console.log("Customer has subscribed before, not eligible for free trial")
+            subscriptionData = {}
+        } else {
+            console.log("New subscriber, creating subscription with free trial")
+            subscriptionData = {trial_period_days: 30}
+        }
```

Comment:
* Never use var
* This can be one line (two if we want logging)
```suggestion
        const subscriptionData = hasCustomerSubscribedBefore(customer) ? {trial_period_days: 30} : {};
        console.log(`Customer is ${subscriptionData.trialPeriodDays ? '' : 'not '}eligible for a free trial.`)
```

Reply:
ah thank you ! i was wondering about how to make it more compact. will do